### PR TITLE
Add support for configuring event listener properties

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

**Comment by arhimondr** [comment]
CC: @linzebing (somewhat I cannot add you as a reviewer)

File: charts/trino/values.yaml [hunk 1]
```diff
@@ -57,6 +57,8 @@ additionalExchangeManagerProperties: {}
 
 additionalCatalogs: {}
 
+eventListenerProperties: {}
```
Comment (hashhar) [inline charts/trino/values.yaml:60]: nit: move above additionalCatalogs to keep all "properties" files together.

**Review by hashhar — APPROVED** [review]
You might need to update README, see output of the `docs` job on CI for what changes are needed.

**Comment by hashhar** [comment]
Can you bump the version as in https://github.com/trinodb/charts/commit/ef74c7c5bedfbc584007cd013f7cdc2ce0957cbc so that a new version is released when this is merged?

**Review by losipiuk — APPROVED** [review]

**Review by linzebing — APPROVED** [review]

**Pull request merged**